evm/src/cpu/kernel/asm/core/jumpdest_analysis.asm

```diff
@@ -282,8 +284,8 @@ global write_table_if_jumpdest:
     // check the remaining path
     %jump(verify_path_and_write_jumpdest_table)
 return:
-    // stack: proof_prefix_addr, jumpdest, ctx, retdest
-    %pop3
+    // stack: packed_opcodes, proof_prefix_addr, ctx, jumpdest, retdest
```

Comment:
nit: the stack order does not match what's in the conditional jump sending us to `return`, i.e.
`jumpdest, ctx, proof_prefix_addr, retdest`.
It doesn't matter as we pop them anyway, but may be better to swap the elements for consistency and clarity.